docs/rnn.rst

```diff
@@ -63,8 +63,8 @@ receives (figure below).
    x_1; x_2; x_3;
 
    node [shape=plaintext];
-   h_0 [label="(0, 0, 0)"]; h_1 [label="(1, 1, 1)"];
-   h_2 [label="(2, 2, 2)"]; h_3 [label="(3, 3, 3)"];
+   h_0 [label="(0, 0, 0)"]; h_1 [label="(2, 2, 2)"];
+   h_2 [label="(4, 4, 4)"]; h_3 [label="(6, 6, 6)"];
```

Comment:
@rizar labels like these. I'm not sure these changes are required, as there is a doctest that verifies that what's currently in these labels corresponds to the output that the code generates.